site/gatsby-site/src/components/landing/Sponsors.js

```diff
@@ -0,0 +1,205 @@
+import React, { useState } from 'react';
+import { Card, Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+import Link from 'components/ui/Link';
+import { Button, Modal } from 'react-bootstrap';
+
+const StyledImageModal = styled.img`
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: space-around;
+  align-items: center;
+  width: 85%;
+  max-width: 355px;
+  max-height: 80px;
+`;
+
+const StyledImage = styled.img`
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: space-around;
+  align-items: center;
+  width: 85%;
+  max-width: 355px;
+  max-height: 80px;
+  cursor: zoom-in;
+`;
+
+const StyledImageCover = styled.img`
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: space-around;
+  align-items: center;
+  max-width: 220px;
+  border: 1px solid black;
+`;
+
+const StyledSubtitle = styled(Card.Subtitle).attrs((props) => ({
+  top: props.top || '30px',
+  bottom: props.bottom || '30px',
+  margin: props.margin || '-.25rem',
+}))`
+  padding-top: ${(props) => props.top};
+  padding-bottom: ${(props) => props.bottom};
+  margin-top: ${(props) => props.margin} !important;
+`;
+
+const SponsorModal = ({ setModalState, modalState, modalName, children, imagePath, title }) => {
+  return (
+    <Modal
+      show={modalState === modalName}
+      onHide={() => setModalState('close')}
+      data-cy="sponsor-modal"
+    >
+      <Modal.Header closeButton>
+        <Modal.Title>{title}</Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        {children}
+        <StyledImageModal src={imagePath} />
+      </Modal.Body>
+      <Modal.Footer>
+        <Button variant="secondary" onClick={() => setModalState('close')} data-cy="close-modal">
+          Close
+        </Button>
+      </Modal.Footer>
+    </Modal>
+  );
+};
+
+export default function Sponsors({ className }) {
+  const [modalState, setModalState] = useState('close');
+
+  return (
+    <Card className={className}>
+      <Card.Body>
+        <Card.Title as="h2">The Responsible AI Collaborative</Card.Title>
+        <Row className="g-0">
+          <Col lg={5} md={12} sm={12}>
+            <Card.Body>
```

Comment:
Having another `<Card.Body>` here makes the text not aligned with the header / cards above.

<img width="300" src="https://user-images.githubusercontent.com/25443411/176262703-3de5f7dd-991d-4207-a7c6-4ebb926980d5.png"/>

If you didn't intend for that to be the case, you could keep the gap between the columns by adding margin-left to the right `<Col>` when it's wide enough to not wrap.